debian/patches/revert-551f560d-cloud-config-after-snap-seeding.patch

```diff
@@ -147,22 +147,21 @@ Last-Update: 2024-02-14
          install = cc.distro.install_packages
  
          for backend, cmd, package in BACKEND_DEF:
-@@ -88,23 +84,21 @@ class TestLxd(t_help.CiTestCase):
-             if backend == "lvm":
-                 self.assertEqual(
-                     [
+@@ -85,22 +81,14 @@ class TestLxd(t_help.CiTestCase):
+                 subp.call_args_list,
+             )
+
+-            if backend == "lvm":
```

Comment:
Is this right?

Reply:
This is... confusing. The original test checks to make sure that the semaphore exists when waiting for snap. But on jammy, I'm not sure why we are still testing for this call. Am I missing something?

Reply:
we were testing the `dm-thin-pool.ko` that was just removed, so I removed that as well.

Reply:
> we were testing the `dm-thin-pool.ko` that was just removed, so I removed that as well.

I know, I'm commenting on the remaining test. I don't think that it tests anything valuable, but I guess it doesn't hurt anything either.